types/office-js/index.d.ts

```diff
@@ -1472,6 +1484,9 @@ declare namespace Office {
     }
     /**
      * Provides information about the environment in which the add-in is running.
+     * 
+     * @remarks
+     * [Api set: ExcelApi 1.1, WordApi 1.1, PowerPointApi 1.1, Mailbox 1.5]
      */
     interface ContextInformation {
         /**
```

Comment:
We should also update Office.Mailbox.diagnostics property and Office.Diagnostics object to note that Office.context.diagnostics is an option available from 1.5. We could use your text "Starting with Mailbox requirement set 1.5, you can also use the {@link https://docs.microsoft.com/javascript/api/office/office.context?view=outlook-js-1.5&preserve-view=true#office-office-context-diagnostics-member | Office.context.diagnostics} property to get similar information." for these.